package.json

```diff
@@ -10,7 +10,7 @@
     "@craco/craco": "^5.6.4",
     "ant-design-pro": "^2.3.2",
     "antd": "^4.16.6",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
```

Comment:
This is a release version change that should be safe, however, if you are going to make changes to any dependencies please explain the regression testing you did on the entire app to confirm it doesn't break anything. 

Release versions were originally supposed to be internal versions before a minor version goes public but OSS has made formal rules like this less real. Always take care when changing a dependency.

Reply:
@tinomen Thank you for the comment. We will definitely make note of it next time. This showed up automatically when we did "NPM I" and assumed it was from the main branch or a prerequisite. What would you recommend doing next time something like this shows up?

Reply:
Correct, the release version was changed by `npm install` because the version string has a carrot `^`. When you see that this happens, run the app as it is and play with it to make sure nothing appears broken. Then commit that change on its own with the commit message explaining what happened. Also, explain your testing in the PR once you open it up for review.

Reply:
Please update the PR description to include the npm updates that occurred during install.